var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -8,117 +8,131 @@
 import os
 import sys
 
-MACOS_VERSION = macos_version() if sys.platform == 'darwin' else None
```

Comment:
@rubendibattista are the `qt` changes intentional? If so, could you undo style changes like exchanging `'` with `"`? It can make it hard to isolate the relevant changes. If not I think something might have occurred when you were merging/rebasing.

Reply:
Sorry, I have automatic linting in my vim with `black` and I forgot to disable it. I just pushed the checked-out version from `upstream/develop`.